plugin/python_dependency_usage/python_dependency_usage.go

```diff
@@ -0,0 +1,191 @@
+package python_dependency_usage
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	"github.com/safedep/code/core"
+	sitter "github.com/smacker/go-tree-sitter"
+)
+
+type DependencyUsagePlugin struct{}
```

Comment:
```suggestion
type dependencyUsagePlugin struct{}
```

Lets not expose internal types outside the package. Instead use a factory / constructor like `func NewDependencyUsagePlugin(...)`

Reply:
ack